models/models.go

```diff
@@ -108,14 +108,14 @@ type CellFormat struct {
 func CreateRenderRequest(reader io.Reader) (*RenderRequest, error) {
 	bytes, err := ioutil.ReadAll(reader)
 	if err != nil {
-		log.Error(err, log.Data{"request_body": string(bytes)})
+		log.Event(nil, "error reading request body", log.ERROR, log.Error(err))
```

Comment:
Should Context be passed down to all of these functions? I've not done so on the basis that it isn't required, and I didn't want to add unnecessary arguments, but appreciate I may be missing something

Reply:
In the other services, we passed the context to all the logs, all the contests come from the main function, or from a request struct that we received.
In this function, I would add a context parameter (first argument), and then use it in the log. But I don't know if there is a guideline for this...

Reply:
This would also apply to other methods that currently have nil context.